app/components/widgets/steps-indicator.js

```diff
@@ -33,5 +33,9 @@ export default Component.extend({
       }
       return step;
     });
-  })
+  }),
+
+  didInsertElement() {
+    this.notifyPropertyChange('autoSteps');
```

Comment:
Call the super method using `this._super();` before anything else inside the `didInsertElement` function